eth/downloader/downloader.go

```diff
@@ -260,6 +281,11 @@ func (d *Downloader) RegisterPeer(id string, version int, currentHead currentHea
 	return nil
 }
 
+// RegisterLightPeer injects a light client peer
```

Comment:
Please expand a bit mentioning it does a wrap. And please end the doc with a punctuation mark :P